### PR TITLE
One page scraper

### PR DESCRIPTION
**Pull request by dreardon1021**
Type of change made:
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [x]  New feature (non-breaking change which adds functionality)
 - [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ]  Refactoring- no new features
 - [ ]  Testing

Description:
Create a webscraper for 1 page of the desired URL

Related Issues/Tickets:
Create WebScraper #11

Files modified:
chromedriver
covents_data.csv
covents_json.json
csv-to-json.py
ebrite-scraper.py

Test Results:
N/A

Notes:
Looking into scraping more pages and have it dynamically function for all available pages for current and following month

### CODE REVIEW

File: webscrapers/covents_json.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
+{
+  "1": {
```
Comment (KellyIB) [inline webscrapers/covents_json.json:2]: Hi, are you storing these id's anywhere?  I think when we add these objects into the DB, the DB will assign them for us, but if these are unique id's that we've pulled from somewhere, they would be useful to store in the DB as a check to see if it already exists before we add them.  If they are not unique (same 20 ids are created each time we scrape, then they might not be necessary, however, I don't think they do any harm to have them.

File: webscrapers/covents_json.json [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    "id": "1",
+    "event_name": "Zayn Malik Party",
+    "event_image": "https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F100189414%2F444571810080%2F1%2Foriginal.20200505-023120?w=500&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C125%2C500%2C250&s=b8e2f287f2ff81f05f5686af7315a584",
+    "event_date_time": "Sun, May 31, 2020 6:00 PM MDT",
```
Comment (KellyIB) [inline webscrapers/covents_json.json:6]: This looks great!, we can break the date and time up for the FE as we store the values in the DB.  Awesome job, Dan!

File: webscrapers/csv-to-json.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import csv, json
+
```
Comment (KellyIB) [inline webscrapers/csv-to-json.py:2]: nice!

File: webscrapers/covents_json.json (hunk 2 above)
Comment (megan-venetianer) [reply webscrapers/covents_json.json:6]: ^ that would be great, Kelly! Agreed, this looks awesome!

**Comment by megan-venetianer** [comment]
I don't understand all of the technical details/syntax of what is going on here, but the data you got out of this looks perfect! Good work!

File: webscrapers/covents_json.json (hunk 1 above)
Comment (dreardon1021) [reply webscrapers/covents_json.json:2]: So I can remove the ids if we want. I can make the keys of each object a number or the event name. I figured the incrementing number is easier to read as some event names are huge. What are your thoughts FE and Kelly

Comment (KellyIB) [reply webscrapers/covents_json.json:2]: Well, the database will assign those id's, but it doesn't hurt to have them in here.

Comment (megan-venetianer) [reply webscrapers/covents_json.json:2]: Ok sounds good!

**Pull request merged**